shimmingtoolbox/coils/spherical_harmonics.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*
+
+import numpy as np
+import math
+from scipy.special import factorial2
+from scipy.special import lpmv
```

Comment:
Wider scope: on my installation, I'm getting "No name 'lpmv' in module 'scipy.special'". I either made an installation error or this isn't included in the installation packages

Reply:
From what I see, indeed there is no lpvm function or object in scipy.special at [https://docs.scipy.org/doc/scipy-1.5.0/reference/special.html](url). @po09i could you give us your scipy version installed?

Reply:
Actually, I do see it -- https://docs.scipy.org/doc/scipy-1.5.0/reference/generated/scipy.special.lpmv.html#scipy.special.lpmv 

I can install this locally, but it would also be good to make sure this is included in the installation dependencies.